frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/FieldListOption.tsx

```diff
@@ -184,6 +184,11 @@ export const BasicFieldOption = forwardRef<BasicFieldOptionProps, 'button'>(
             <Text noOfLines={1}>Use for approvals</Text>
           </Badge>
         ) : null}
+        {fieldType === BasicField.Signature ? (
```

Comment:
Can use a marker so that it is easier to look up todos associated with a particular feature!
```suggestion
        ) : null}
        {/* TODO: FRM-2054 remove when signature field is out of beta */}
        {fieldType === BasicField.Signature ? (
```